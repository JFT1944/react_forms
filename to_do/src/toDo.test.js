
import React from "react";
import { render, fireEvent, getByTestId } from "@testing-library/react";
import  ToDo from "./toDo";
// import TEST_IMAGES from "./_testCommon.js";
import App from './App'



it("renders without crashing", async function() {
    render(<ToDo  />);
  });


it("matches snapshot", function() {
    const {asFragment} = render(<ToDo />);
    expect(asFragment()).toMatchSnapshot();
  });
  

// Tests to make, then turn in.
// it('should create a new square', function(){
//     let {getByTestId, getByText} = render(<ToDo />)
//     let submitB = getByTestId('subBut')
//     fireEvent.click('submitB')
// })