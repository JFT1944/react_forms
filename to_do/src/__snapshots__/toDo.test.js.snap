// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <li>
     
    <button
      data-testid="subButundefined"
    >
      X
    </button>
  </li>
</DocumentFragment>
`;
